js/templates/buyWizard.html

```diff
@@ -28,10 +28,12 @@ <h3 class="custCol-text padding30"><%= polyglot.t('CheckExchanges') %> </h3>
         </div>
         <div class="txt-bright pad20">
           <div class="txt-center row10 textSize20px custCol-text">
-            <%= polyglot.t('buyFlow.SendBTCtoAddress', {amount: '<span class="js-buyWizardDetailsTotalBTC"></span>'}) %>
-          </div>
-          <div class="txt-center row10">
-            <a class="textSize16px js-buyWizardPayLink custCol-text"><span class="js-buyWizardPayURL"></span></a>
+            <%=
+              polyglot.t('buyFlow.SendBTCtoAddress', {
+                amount: '<span class="js-buyWizardDetailsTotalBTC"></span>',
+                recipient: '<a class="textSize16px js-buyWizardPayLink custCol-text"><span class="js-buyWizardPayURL"></span></a>'
```

Comment:
The recipient is a link styled to occupy it's own line under the amount. If they move it to be in the middle of the sentence, it could cause some issues. 

Example:

![image](https://cloud.githubusercontent.com/assets/1584275/15230613/90e6491c-1864-11e6-8e62-61cb39965ea5.png)